python/README.md

```diff
@@ -50,21 +50,39 @@ Installation
 4) Build and run the tests:
 
      $ python setup.py build
-     $ python setup.py google_test
+     $ python setup.py test
 
-     To build the C++ implementation run:
-     $ python setup.py build --cpp_implementation
+     To build, test, and use the C++ implementation, you must first compile
+     libprotobuf.so:
+
+     $ (cd .. && make)
+
+     On OS X:
```

Comment:
I guess what I'm getting at is that it would be nice if the README didn't have to have an entire separate section to address OS X, brew, etc. Your caveat about Brew doesn't apply if the user sets DYLD_LIBRARY_PATH, right? Instead it could just say something like:

      You must make libprotobuf.so dynamically available. You can either
      install libprotobuf you built earlier, or set LD_LIBRARY_PATH
      (DYLD_LIBRARY_PATH on OS X):

      $ export LD_LIBRARY_PATH=../src/.libs
      $ export DYLD_LIBRARY_PATH=../src/.libs     # for OS X
      or
      $ (cd .. && make install)